_posts/grassycheetah94/2017-05-22-thomas-turtle-hack.md

```diff
@@ -0,0 +1,207 @@
+---
+layout: post
+author: grassycheetah94
+title: "Thomas' Turtlehack"
+---
+
+Here is the program I am embedding:
+<iframe src="https://trinket.io/embed/python/2a682241b6" width="100%" height="600" frameborder="0" marginwidth="0" marginheight="0" allowfullscreen></iframe>
+
+---
+Reflection:
+I wanted to create a turtle version of the ‘Hello Chuck’ assignment we had this weekend. I started with creating a ‘man’. This took a lot longer than I originally thought it would, so he is not as detailed as I wanted him to be.  But the basic concept is still there. As you can see below.
```

Comment:
Great idea to pick a previous assignment as a starting point and add a (very detailed!) drawing to it.